### PR TITLE
feat: output full content for rss feed

### PR DESCRIPTION
**Pull request by vimcaw**
I have tested on `examples/theme-leerob` of this repo and anything works well.

The post at `examples/theme-leerob/docs/posts/hello-world.md`:
```md
---
title: "Hello World"
titleImage: "https://img.alicdn.com/imgextra/i2/O1CN01Q93IFG1zkvJLH40c6_!!6000000006753-2-tps-2560-1440.png"
titleImageCaption: "介绍"
publishedAt: "2023/09/24"
---

Happy to see you here! This is a demo post.

```

RSS feed:
```xml
<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
    <id>https://your-site.com/</id>
    <title>Docaid</title>
    <updated>2023-02-13T06:53:29.591Z</updated>
    <generator>https://github.com/jpmonette/feed</generator>
    <author>
        <name>Your Name</name>
        <email>your-email@gmail.com</email>
        <uri>https://your-site.com/</uri>
    </author>
    <link rel="alternate" href="https://your-site.com/"/>
    <link rel="self" href="https://your-site.com/rss.xml"/>
    <subtitle>Docaid</subtitle>
    <logo>https://img.alicdn.com/imgextra/i1/O1CN01FwDVoc1YGhgE28l5B_!!6000000003032-2-tps-200-200.png</logo>
    <icon>https://img.alicdn.com/imgextra/i1/O1CN01FwDVoc1YGhgE28l5B_!!6000000003032-2-tps-200-200.png</icon>
    <rights>Docaid</rights>
    <entry>
        <title type="html"><![CDATA[Hello World]]></title>
        <id>https://your-site.com/posts/hello-world</id>
        <link href="https://your-site.com/posts/hello-world"/>
        <updated>2023-09-23T16:00:00.000Z</updated>
        <summary type="html"><![CDATA[Hello World]]></summary>
        <content type="html"><![CDATA[<p>Happy to see you here! This is a demo post.</p>
]]></content>
    </entry>
</feed>
```

Reeder 5 mac app:
CleanShot 2023-02-13 at 14 55 34

### CODE REVIEW

**Comment by vimcaw** [comment]
Just found a bit problem with it, it doesn't respect the `content` field in `transform` options. I will fix it later.

**Comment by vimcaw** [comment]
It looks some inconsistent in your `transform` options:

The `content` field is the source content of Markdown here:
https://github.com/sorrycc/docaid/blob/78bf26e1f5f8ad9d0bf9b5e8a21600b7fb108f21/docaid/src/utils/parseDocs.ts#L93-L104

But you use it to transform to HTML content here:
https://github.com/sorrycc/weekly/blob/54494dc297e22503301318b8ceb8a8d2f763cebe/.umirc.ts#L74-L84

We should make it consistent, can the `transform` option modify the source content of the Markdown file? Or modify output only? (In this case, we should use the `html` field instead of the `content` field.)

File: docaid/src/utils/generateRSS.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export function generateRSS(opts: {
       ...item,
       id: item.id || item.link,
       description: item.description || item.title,
+      content: item.html,
```
Comment (sorrycc) [inline docaid/src/utils/generateRSS.ts]:
```suggestion
```

File: umi-plugin-docaid/src/buildDocs.ts [hunk 2]
```diff
@@ -116,6 +116,7 @@ export async function buildDoc(root: string, dir: string, opts: IOpts) {
           link: `${opts.config.siteUrl}${dir}/${doc.file.replace(/\.md$/, '')}`,
           // TODO: support full content by default
           content: '',
+          html: doc.html,
```
Comment (sorrycc) [inline umi-plugin-docaid/src/buildDocs.ts]:
```suggestion
          content: doc.html,
```

**Review by sorrycc — APPROVED** [review]

**Pull request merged**